src/hir/hir.rs

```diff
@@ -22,6 +22,8 @@ pub enum Hir {
     FunctionDefinition(FunctionDefinition),
     /// A comment.
     Comment(Comment),
+    /// An expression.
+    Expression(Expression),
```

Comment:
Hmmm, maybe I'm wrong, but I would expect `vm.skip(true);` to be a statement and not an expression.

Reply:
Correct - I refactored this, check infra for original idea